src/server/trpc.ts

```diff
@@ -0,0 +1,9 @@
+import { initTRPC } from "@trpc/server";
+// Avoid exporting the entire t-object
+// since it's not very descriptive.
+// For instance, the use of a t variable
+// is common in i18n libraries.
+const t = initTRPC.create();
```

Comment:
We should probably use [superjson](https://www.npmjs.com/package/superjson) as the transformer so that dates and other stuff get properly (de)serialized (recommended by `create-t3-app`)

```suggestion
import superjson from "superjson";

const t = initTRPC.create({
  transformer: superjson,
});
```